calcom/cal.com
# fix: Change conferencing app-store URL on event types and bookings pages

## What does this PR do?

- Fixes #15811
- Fixes CAL-4063

/claim #15811

<img width="1026" alt="Screenshot 2024-07-23 at 6 03 14 PM" src="https://github.com/user-attachments/assets/5431e7b7-723d-408b-b304-2e72106d0c3b">


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Go to any event edit page, e.g - `http://localhost:3000/event-types/<event_id>?tabName=setup`
- Scroll to the "Location" section.
- In the text `Can't find the right video app? Visit our App Store`, the "App store" link should be `apps/categories/conferencing`.

## Checklist
